### ISSUE TITLE
6to5 deprecation warning on npm install

### ISSUE THREAD

**Issue by pdehaan** [n1k0/readinglist-client#10]
### Steps to reproduce:

``` sh
$ npm i

npm WARN deprecated 6to5@2.13.7: 3.0.0 has been released featuring a host of bug fixes and new features
```

It looks like the [**karma-6to5-preprocessor**](https://www.npmjs.com/package/karma-6to5-preprocessor) module may be the only outdated module that got installed from a brand new `git clone`:

``` sh
$ npm outdated --depth 0

Package                  Current  Wanted  Latest  Location
karma-6to5-preprocessor    1.0.0   1.0.0   3.0.0  karma-6to5-preprocessor
```

Current [package.json:43](https://github.com/mozilla-services/readinglist-client/blob/b5f46cd43864f856bf31df6e753d317794d64852/package.json#L43) points to `"^1.0.*"`:

``` js
"karma-6to5-preprocessor": "^1.0.*",
```

**Comment by pdehaan** [comment]
Unrelated (and possibly unavoidable) but it looks like karma may be using an older and potentially vuln version of **connect**. I can dig deeper and see if there is a bug and/or pull request out for karma. I'm guessing there isn't a newer version available since I didn't see anything from `npm outdated --depth 0`:

``` sh
$ npm shrinkwrap --dev
wrote npm-shrinkwrap.json

$ nsp shrinkwrap
Name          Installed  Patched  Vulnerable Dependency
serve-static    1.6.4    >=1.7.2  readinglist-client > karma > connect
```

---

**UPDATE:** Filed upstream as https://github.com/karma-runner/karma/issues/1295

**Comment by n1k0** [comment]
@pdehaan tbh I'm currently considering moving away from karma, as its local server seems to be fairly unstable, and its configuration is kinda painful with this project setup; I'll investigate [testem](https://github.com/airportyh/testem) and [mochify](https://www.npmjs.com/package/mochify) as a replacement, though it's not P1 for now.

In the meanwhile, I'll update the `karma-6to5-preprocessor` package. Thanks for the report.

**Comment by n1k0** [comment]
Fixed by ff1a53623721fd42237da6462c18e4ba5ebc3c68 along with #10 & #15.
